cloudinit/userdata_cloudinit.erb

```diff
@@ -41,6 +49,28 @@ users:
    lock-passwd: false
    passwd: <%= @host.root_pass %>
 
+ runcmd:
+ <% if rhel_compatible -%>
+ - <%= snippet('epel') -%>
```

Comment:
http://yaml.org/
I prefer no indentation on arrays like in ```Projects:```
However, if you prefer, I can switch all arrays in this template to 2 spaces like in ```Related Projects:```

An indent block is not necessary, since the epel template only returns one line.  I'm chomping the newline by using ```-%>```